README.md

```diff
@@ -1,8 +1,10 @@
+found a typo issue file
```

Comment:
This should be in the pull request message and should be removed from the README

Reply:
ok thanks thehappydinoa will do like wise